proposals/lip-0043.md

```diff
@@ -480,6 +477,8 @@ def execute(trs: Transaction) -> None:
             },
             "mainchainValidators": {
                 "type": "array",
+                "minItems": 1,
+                "maxItems": NUMBER_ACTIVE_VALIDATORS_MAINCHAIN,
```

Comment:
These requirements are redundant because the checks are already performed in the `verify` hook. And if the checks are repeated in the execute (if we should decided that encode should always perform full schema validation) than we do some unnecessary execution steps.
The same holds for the lines 474-475 and 489 (I also missed this before).

Reply:
discussed in person, we keep it in the schema

Reply:
I got convinced that there good reasons to keep it.